parlai/scripts/train_model.py

```diff
@@ -255,7 +257,13 @@ def log(self):
 
         # time elapsed
         logs.append('time:{}s'.format(math.floor(self.train_time.time())))
-        logs.append('total_exs:{}'.format(self.world.get_total_exs()))
+        total_exs = self.world.get_total_exs()
+        logs.append('total_exs:{}'.format(total_exs))
+
+        exs_per_ep = self.world.num_examples()
+        if exs_per_ep:
+            logs.append('total_epochs:{}'.format(
+                round(total_exs / exs_per_ep, 2)))
```

Comment:
oops this is extra and should be merged already in a slightly modified form, ignore this section